examples/templates/js/src/styles.css

```diff
@@ -0,0 +1,11 @@
+body {
+    padding: 1rem;
+    margin: 0;
+}
+
+#wrapper {
+    overflow: auto;
+    min-width: 100%;
+    min-height: 100%;
+    height: calc(100vh - 2rem);
+}
```

Comment:
Note to self: check if this is right for all use case scenarios

Reply:
See: https://github.com/handsontable/handsontable/pull/7588#pullrequestreview-612535840